app/controllers/users_controller.rb

```diff
@@ -0,0 +1,20 @@
+class UsersController < ApplicationController
+  def index
+  end
+  def create
+    @user = params[:user]
+    if User.find_by(email: @user[:email], password: @user[:password])
+      @user_new = User.find_by(email: @user[:email])
+      session[:user_email] = @user[:email]
+      session[:user_id] = @user_new.id
+      redirect_to '/events'
+    else
+      render plain: 'Error'
```

Comment:
Show proper error instead of general error.

Like email doesn't exists or password doesn't match.